tasks/edit_block/__init__.py

```
@@ -20,23 +21,26 @@ class Outputs(typing.TypedDict):
-    3. Fetch result and download image
-    """
-    
-    # Get API configuration from environment
     console_api_url = context.oomol_llm_env.get("base_url")
     api_key: Any = context.oomol_llm_env.get("api_key")
     
-    file_path = params["file"]
+    file_paths = params["files"]
     prompt = params["prompt"]
+    model = params.get("model", "nano-banana/edit")
+    output_dir = params.get("output_dir", "/oomol-driver/oomol-storage")
```

Ruff (0.12.2)

Local variable `output_dir` is assigned to but never used

Remove assignment to unused variable `output_dir`

(F841)